### DISCUSSION TITLE
change the default size of rich text box

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2057] [General]
How can I change the default size of the rich text box I would like it much longer. I know you can drag it longer i want the default size to me longer  
![image](https://user-images.githubusercontent.com/80058238/126540429-55b4b512-b872-4ae8-b317-48a934dacd59.png)

**Comment by joebudi** [comment]
@AndrewJRamay this is not possible right now. I'm not sure it is specifically an issue, but it is something I have raised with the team. A specific height setting would suffice.

**Reply by KernOil** [reply]
@joebudi  Thank you!
